src/Api/Domain.php

```diff
@@ -88,16 +88,26 @@ public function create(string $domain, string $smtpPass = null, string $spamActi
 
         $params['name'] = $domain;
 
-        // If at least smtpPass available, check for the fields spamAction wildcard
         if (!empty($smtpPass)) {
+            Assert::stringNotEmpty($smtpPass);
+            Assert::lengthBetween($smtpPass, 5, 32, 'SMTP password must be between 5 and 32 characters.');
```

Comment:
Would you mind please removing this Assert? Even if at the moment the API docs are setting this requirement, we have no warranties that it will be kept in the future. It makes more sense to allow the API itself to return an error message regarding this issue so that the validation will always happen on their end.